hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStopSubcommand.java

```diff
@@ -34,6 +34,7 @@
 public class ContainerBalancerStopSubcommand extends ScmSubcommand {
   @Override
   public void execute(ScmClient scmClient) throws IOException {
+    System.out.println("ContainerBalancer is in stopping state. Waiting for current iteration to finish");
     scmClient.stopContainerBalancer();
     System.out.println("Stopping ContainerBalancer...");
```

Comment:
`...` indicates the command is doing something (waiting for the server).  I think it should appear before sending the stop command.

```
ContainerBalancer is in stopping state. Waiting for current iteration to finish...
ContainerBalancer stopped.
```

or even (printed in two steps):

```
Stopping ContainerBalancer... done.
```

Reply:
> `...` indicates the command is doing something (waiting for the server). I think it should appear before sending the stop command.
> 
> ```
> ContainerBalancer is in stopping state. Waiting for current iteration to finish...
> ContainerBalancer stopped.
> ```

+1

Reply:
NIT: Maybe I'm being overly technical, but how about:
```
Sending stop command. Waiting for Container Balancer to stop...
```
and then 
```
Container Balancer stopped.
```
Balancer doesn't transition to the `STOPPING` state until it has received the stop command, done some validations (this scm is the leader, balancer is currently `RUNNING` etc) and persisted some configurations to RocksDB.